models/labels/daos/labels_daos_schema.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+
+models:
+  - name: labels_daos
```

Comment:
I think adding a uniqueness test to make sure there are no duplicates would be great

Reply:
Thank you ser @soispoke, I'll update and add tests and also change the query source to `query`. 

Also, I can definitely do the pr to update and change from `daos` to `dao`, I have another pr open right now (to add polygon wallets & transactions) and I'm using `daos`. 

Should I submit that **pr** after these pr's are merged? or should I submit a name change to all the models along with this pr? and update my other pr as well?